crates/uv-cli/src/lib.rs

```diff
@@ -534,8 +534,19 @@ pub struct VersionArgs {
     pub value: Option<String>,
 
     /// Update the project version using the given semantics
+    ///
+    /// This flag can be passed multiple times, and the bumps will be applied in the
+    /// following order that prevents bumps from being undone by other bumps:
+    ///
+    ///     major > minor > patch > stable > alpha > beta > rc > post > dev
+    ///
+    /// When using the prerelease modes (alpha, beta, rc), usage will typically look like:
+    ///
+    /// 1. ship a beta of the next release: `uv version --bump patch --bump beta`
+    /// 2. ship another beta:               `uv version --bump beta`
+    /// 3. ship the stable version:         `uv version --bump stable`
```

Comment:
nit: This is rendered as markdown, the whitespace alignment will be lost.

Reply:
Ah yeah I wanted it to render helpfully in the CLI but I think I need to move these docs off `bump` and onto the command itself...

Reply:
...no we supress verbose help so much it's web docs or nothing, oh well!

Reply:
we also fix some markdown parsing in https://github.com/astral-sh/uv/pull/13166

Reply:
![Screenshot 2025-05-21 at 4 28 37 PM](https://github.com/user-attachments/assets/6b301e0d-14a8-4ebc-80fc-2271ef261a51)

i'm being pranked by pseudo markdown 😭

Reply:
:/ another case of https://github.com/clap-rs/clap/issues/5900